src/services/news.data.service.ts

```diff
@@ -39,10 +43,17 @@ export function fetchNews(source = '', count = 5): Promise<any> {
   // return mockHTTPCall<Array<INewsFlash | any>>(newsFlashCollectionData);
   return axios
     .get(url)
-    .then((res) => res.data)
+    .then((res) => addDemoNewsflash(res.data))
     .catch(onErrorFetchNewsFlash);
 }
 
+function addDemoNewsflash(data: Array<any>) {
```

Comment:
* add a comment here
* maybe we should move all mocking to a separate service?...